tests/base/quadrature_test.output

```diff
@@ -83,6 +131,22 @@ DEAL:2d:subfaces::Quadrature no.17 is exact for polynomials of degree 5
 DEAL:2d:subfaces::Quadrature no.18 is exact for polynomials of degree 7
 DEAL:2d:subfaces::Quadrature no.19 is exact for polynomials of degree 9
 DEAL:2d:subfaces::Quadrature no.20 is exact for polynomials of degree 11
+DEAL:2d:subfaces::Quadrature no.21 is exact for polynomials of degree 0
+DEAL:2d:subfaces::Quadrature no.22 is exact for polynomials of degree 0
+DEAL:2d:subfaces::Quadrature no.23 is exact for polynomials of degree 2
+DEAL:2d:subfaces::Quadrature no.24 is exact for polynomials of degree 2
+DEAL:2d:subfaces::Quadrature no.25 is exact for polynomials of degree 4
+DEAL:2d:subfaces::Quadrature no.26 is exact for polynomials of degree 4
+DEAL:2d:subfaces::Quadrature no.27 is exact for polynomials of degree 6
+DEAL:2d:subfaces::Quadrature no.28 is exact for polynomials of degree 6
+DEAL:2d:subfaces::Quadrature no.29 is exact for polynomials of degree 8
+DEAL:2d:subfaces::Quadrature no.30 is exact for polynomials of degree 8
```

Comment:
@jpthiele  Looks correct now!?

Reply:
Yes, the quadrature formulas are indeed correct now.
Interestingly the final few digits seem to have had quite a large impact.